automated install/basic-install.sh

```diff
@@ -267,6 +267,51 @@ setStaticIPv4() {
 	fi
 }
 
+setDNS(){
+	DNSChoseCmd=(whiptail --separate-output --radiolist "Select DNS Servers" $r $c 2)
+	DNSChooseOptions=(Google "Use Google's DNS Servers" on
+					  DynDNS "Use DynDNS's DNS Servers" off)
+	DNSchoices=$("${DNSChoseCmd[@]}" "${DNSChooseOptions[@]}" 2>&1 >/dev/tty)
+
+	case $DNSchoices in
+		Google)
+			echo "Google selected."
+			piholeDNS1="8.8.8.8"
+			piholeDNS2="8.8.4.4"
+			;;
+		DynDNS)
+			echo "DynDNS selected."
+			piholeDNS1="208.67.222.222"
+			piholeDNS2="208.67.220.220"
+			;;
+	esac
+}
+
+versionCheckDNSmasq(){
+	# Check if /etc/dnsmasq.conf is from pihole.  If so replace with an original and install new in .d directory
+	dnsFile1="/etc/dnsmasq.conf"
+	dnsFile2="/etc/dnsmasq.conf.orig"
+	dnsSearch="addn-hosts=/etc/pihole/gravity.list"
+	
+	# Check dnsmasq.conf for pihole magic
+	if grep -q $dnsSearch $dnsFile1; then
```

Comment:
'grep -q` throws an error when a file is non-existant. Think this needs a `grep -s`?